app/main.py

```
@@ -1,6 +1,54 @@
+import os
+import sys
+from datetime import datetime
+
+import requests
+from dotenv import load_dotenv
+
+load_dotenv()
+
+API_KEY = os.environ.get("API_KEY")
+if not API_KEY:
+    print("API_KEY is missing. Please set API_KEY.")
+    sys.exit(1)
```

You can raise an Error with this message